examples/kuesa/music-box/qml/MainScene.qml

```diff
@@ -0,0 +1,331 @@
+import Qt3D.Core 2.10
+import Qt3D.Render 2.11
```

Comment:
Might make sense to use include 2.12 everywhere as we assume Qt 5.12 for Kuesa ?